source/scenes/home/Asset/Asset/Asset.container.tsx

```diff
@@ -65,20 +67,25 @@ const AssetDetailContainer = ({ navigation }: IAssetDetail) => {
     });
   }, []);
 
-  const fetchTxs = async () => {
-    if (activeAsset.type === AssetType.Constellation) {
-      return activeAsset.transactions;
-    }
-    return await accountController.getFullETHTxs().sort((a, b) => b.timestamp - a.timestamp);
-  };
+  useEffect(() => {
+    const fetchTxs = async () => {
```

Comment:
Can the fetch transactions be moved to a thunk?

Reply:
Yes, but let's leave that for future optimization since that would mean pulling all the asset txns from redux and that would have other implications.